sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -82,6 +82,7 @@
  */
 public final class BatchSpanProcessor implements SpanProcessor {
 
+  // FIXME This worker thread can be eliminated by leveraging the timer
   private static final String WORKER_THREAD_NAME =
```

Comment:
I just noticed that the worker thread can now be eliminated as we can leverage the timer thread. I'll do this in a separate PR if you accept this one. ;-)